spec/controllers/admin/reports/registered_adviser_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Admin::Reports::RegisteredAdviserController, type: :controller do
+  describe '#filename' do
+    before(:each) do
+      Timecop.freeze(date)
+    end
+
+    after(:each) do
+      Timecop.return
+    end
+
+    context 'yesterday' do
```

Comment:
♻️  @norm && @willhall88 this has been fixed here: https://github.com/moneyadviceservice/rad/pull/406/commits/5e40bb7d84f3a5b24f1f6f41bfd070ff95b970e2